server/main.go

```diff
@@ -3,13 +3,15 @@ package main
 import (
 	"log"
 	"net/http"
+
+	gitop "github.com/litmuschaos/charthub.litmuschaos.io/server/pkg/gitops"
+	routes "github.com/litmuschaos/charthub.litmuschaos.io/server/routes"
```

Comment:
No need to specify the variable here, If you are using the same name as the package name

Reply:
Ok done!